src/raiden_libs/matrix.py

```diff
@@ -170,6 +170,7 @@ def follow_address_presence(self, address: Address, refresh: bool = False) -> No
             self.user_manager.add_address(address)
 
             if refresh:
+                self.user_manager.populate_userids_for_address(address)
```

Comment:
Does this need an extra method? Shouldnt `refresh_address_presence` populate the userids if necessary? Or is there no way to detect that an update is necessary, so we have to call this explicitly?

Reply:
Not sure why @ulope went that path, let's see what he says.